src/components/layout/AppBar.js

```diff
@@ -154,12 +156,12 @@ function DEAppBar(props) {
     }, [ref, setAppBarRef]);
 
     useEffect(() => {
-        if (userProfile) {
+        if (userProfile && !bootstrapInfo) {
             setBootstrapQueryEnabled(true);
         }
-    }, [userProfile]);
+    }, [userProfile, bootstrapInfo]);
 
-    useBootStrap(bootstrapQueryEnabled, null, setBootstrapError);
+    useBootStrap(bootstrapQueryEnabled, setBootstrapInfo, setBootstrapError);
```

Comment:
While we had bootstrap context, we never put it to full use by setting it up when bootstrap was fetched from services. This will fix it.